pkg/builders/buildpacks/builder_test.go

```diff
@@ -116,6 +119,46 @@ func TestBuild_BuilderImageConfigurable(t *testing.T) {
 	}
 }
 
+// TestBuild_BuilderImageExclude ensures that ignored files are not added to the func
+// image
+func TestBuild_BuilderImageExclude(t *testing.T) {
+	var (
+		i = &mockImpl{} // mock underlying implementation
+		b = NewBuilder( // Func Builder logic
+			WithName(builders.Pack), WithImpl(i))
+		f = fn.Function{
+			Runtime: "go",
```

Comment:
You can use something like filepath.Join(t.TempDir(), "fn") as `Root`. Then directory removal will done automatically no need for defers.